code/modules/vehicles/armored/interiors/chairs.dm

```diff
@@ -78,10 +78,17 @@
 
 /obj/structure/bed/chair/vehicle_crew/driver/post_buckle_mob(mob/buckling_mob)
 	. = ..()
+	buckling_mob.reset_perspective(owner)
+	ADD_TRAIT(buckling_mob, TRAIT_SEE_IN_DARK, VEHICLE_TRAIT)
+	buckling_mob.update_sight()
+	buckling_mob.client.view_size.set_view_radius_to(4.5)
```

Comment:
Remember I did the vis range mod on the vehicle, which applies in /obj/structure/bed/chair/vehicle_crew/post_buckle_mob

If you're giving drivers this zoom level, maybe its better to change that vehicle var back to a chair var?